gmso/tests/test_forcefield_xml.py

```diff
@@ -136,6 +138,12 @@ def test_elementary_charge_to_coulomb(self, ff):
         elementary_charge = ff.atom_types['Li'].charge.to(u.elementary_charge)
         assert elementary_charge.units == u.Unit(u.elementary_charge)
 
+    def test_write_xml(self, ff):
+        dir_name = mkdtemp()
```

Comment:
https://docs.python.org/3/library/tempfile.html#tempfile.mkdtemp
> The user of mkdtemp() is responsible for deleting the temporary directory and its contents when done with it.

I strongly recommend removing `mkdtemp` and using `tempfile`'s context managers for tests that need to write files.
https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory
e.g. `with TemporaryDirectory() as tempdir: ...`